extensions/wikia/Recirculation/RecirculationApiController.class.php

```diff
@@ -15,109 +14,20 @@ public function __construct() {
 		$this->cors->setAllowAllOrigins();
 	}
 
-	public function getFandomPosts() {
-		$this->cors->setHeaders( $this->response );
-
-		$type = $this->getParamType();
-		$cityId = $this->getParamCityId();
-		$limit = $this->getParamLimit();
-		$fill = $this->getParamFill();
-		$slug = $this->getParamSlug();
-
-		$title = wfMessage( 'recirculation-fandom-title' )->plain();
-		$dataService = $this->getDataServiceInstance($type, $cityId);
-		$posts = $dataService->getPosts( $type === 'stories' ? $slug : $type, $limit );
-
-		if ( $fill === 'true' && count( $posts ) < $limit ) {
-			$ds = new ParselyDataService( $cityId );
-			$posts = array_slice( array_merge( $posts, $ds->getPosts( 'recent_popular', $limit ) ), 0, $limit );
-		}
-
-		$this->response->setCacheValidity( WikiaResponse::CACHE_STANDARD );
-		$this->response->setData( [
-			'title' => $title,
-			'posts' => $posts,
-		] );
-	}
-
 	public function getPopularWikiArticles() {
```

Comment:
I can not find any usages of this method, can we get rid of it?

Reply:
For the time being it is used by mobile-wiki at https://github.com/Wikia/mobile-wiki/blob/dev/app/components/recirculation-prefooter.js#L100, we can definitely remove it after we update mobile-wiki